src/applications/toe/reducers.js

```diff
@@ -24,6 +27,36 @@ export default {
   form: createSaveInProgressFormReducer(formConfig),
   data: (state = initialState, action) => {
     switch (action.type) {
+      case FETCH_PERSONAL_INFORMATION:
+        return {
+          ...state,
+          personalInfoFetchInProgress: true,
+        };
+      case FETCH_PERSONAL_INFORMATION_SUCCESS:
+      case FETCH_PERSONAL_INFORMATION_FAILED:
+        return {
+          ...state,
+          personalInfoFetchComplete: true,
+          personalInfoFetchInProgress: false,
+          formData: action?.response || {},
+          fetchedSponsorsComplete: true,
+          sponsors: {
+            sponsors: [
+              ...action?.response?.data?.attributes?.toeSponsors?.transferOfEntitlements.map(
+                (sponsor, index) => {
+                  return {
+                    ...sponsor,
+                    // ! TODO: CHANGE ID
```

Comment:
Think you forgot to revert this change for testing.

Reply:
Should that be reverted after testing? At the moment, the api's id's are all the same.